### ISSUE TITLE
Organic Chemistry - webview - Answer Key -- align distractor letter to answer, distractor letter empty link

### ISSUE THREAD

**Issue by ww-openstax** [openstax/cnx-recipes#5207]
# Brief Description of Scenario
In WebView Answer Key:
1) align distractor letter with answer
2) distractor letter is formatted as link but it is an empty link
Related to https://github.com/openstax/cnx-recipes/issues/5185

![image](https://github.com/openstax/cnx-recipes/assets/89537088/d001f9e1-f3cf-4790-9f7d-cd7056519a91)

# Pattern of Occurrence 

# Acceptance Criteria

  
# Sample Mark-Up
```
<!-- Please paste here if applicable. -->
```

# Location(s)
- **osbooks repo:** osbooks-organic-chemistry
- **collection slug:** organic-chemistry
- **ref/branch:** 
- **chapter/section:** webview Answer Key 
- **surrounding text:**

**Comment by wilkus27** [comment]
@lwtchu This is what it looks like

![image](https://github.com/openstax/cnx-recipes/assets/40228854/67537819-6812-4e3b-a643-a01328383962)

Images have class `scaled-down30` and they appear very small now.

This is what it looks like without decreasing width to 30%

![image](https://github.com/openstax/cnx-recipes/assets/40228854/49307c58-8ad2-4cf3-a537-4a580b55fdc0)

**Comment by ww-openstax** [comment]
@lwtchu WiseWire noticed the comment above about scaling of images. WiseWire is currently in the process of reviewing the webview and adjusting the scaling of images for all chapters. 
Are there any OSX code fixes in-process that will affect the output of scaled images?

**Comment by lwtchu** [comment]
@ww-openstax There aren't any code fixes that will affect the output of scaled images, we just weren't sure if it was missed. You can continue the review :)

**Comment by Stackblocks** [comment]
Answers are now inline with the letter and the letters are no longer links, but the answers are also no longer stacked vertically and are fully inline. Is this expected? @lwtchu 

![CleanShot 2023-08-09 at 18.18.18.png](https://images.zenhubusercontent.com/5af1f4c24b5806bc2bd257f2/41bd4e7c-feb5-49b3-8376-0beb8d78506c)

**Comment by lwtchu** [comment]
@Stackblocks I think it's okay for now -- @wilkus27 is the line wrapping something done specific in the webview for multi part questions? Trying to understand how this impacts the entire library since they share a stylesheet.

**Comment by wilkus27** [comment]
@lwtchu Yes, I tried to make a change that only affects multi part questions. There is only one property that I'm not fully sure about. I changed width for `os-solution-container` from `[question-solution]` from `100%` to `auto`. So it might have an impact on Answer Key solutions from injected exercises. But we don't have any other styles for injected exercises solutions for now. They are not styled on webview. They don't look bad because they inherit some properties from other elements. In general we have a little mess in webview styles :see_no_evil: I think we need to start to refactor it in the near future.
